web-registry/package.json

```diff
@@ -62,9 +68,8 @@
     "lint-fix": "eslint ./src --fix -c .eslintrc.js --ext .ts,.tsx",
     "format": "prettier --write --loglevel warn './src/**/*.{ts,tsx,json,md,css}'",
     "format-and-fix": "yarn format && yarn lint-fix",
-    "start": "react-scripts start",
-    "build": "react-scripts build",
-    "test": "react-scripts test",
+    "start": "react-app-rewired start",
```

Comment:
curious to know what this change is for?

Reply:
Add to use that to fix an alias issue. This script allows us to override the webpack alias config, you can see it here: https://github.com/regen-network/regen-web/pull/1514/files#diff-7948d3454e9336c2f9cccf4385651c122ee3940f1b4403030971e1d63fe609dd